tests/test_autonomy/test_cli/test_scaffold_fsm.py

```diff
@@ -67,72 +68,47 @@ def setup_class(cls) -> None:
 
         cls.author = get_default_author_from_cli_config()
 
-    @classmethod
     def setup(
-        cls,
+        self,
     ) -> None:
         """Setup test."""
 
-        cls._t = TemporaryDirectory()
-        cls.t = Path(cls._t.name)
-        cls.packages_dir_path = str(cls.t / PACKAGES)
-
-        shutil.copytree(ROOT_DIR / PACKAGES, cls.packages_dir_path)
-
-        cls.cwd = Path.cwd()
-        os.chdir(cls.t)
-
-        cls.run_cli_command(
-            "--registry-path",
-            str(cls.packages_dir_path),
+        self.run_cli_command(
             "create",
             "--local",
             "--empty",
-            cls.agent_name,
+            self.agent_name,
             "--author",
-            cls.author,
+            self.author,
         )
 
+    def teardown(
+        self,
+    ) -> None:
+        """Teardown test."""
+        with suppress(OSError, FileExistsError, PermissionError):
+            shutil.rmtree(str(Path(self.t, self.agent_name)))
+
     def scaffold_fsm(self, fsm_spec_file: Path) -> click.testing.Result:
         """Scaffold FSM."""
 
         *_, skill_name, _ = fsm_spec_file.parts
         skill_name = f"test_skill_{skill_name}"
         scaffold_args = ["scaffold", "fsm", skill_name, "--local", "--spec"]
         cli_args = [
-            "--registry-path",
-            self.packages_dir_path,
             *scaffold_args,
             fsm_spec_file,
         ]
         scaffold_result = self.run_cli_command(*cli_args, cwd=self.t / self.agent_name)
-        push_result = self.run_cli_command(
```

Comment:
no idea why this was needed here...

Reply:
if we push to registry then we need to setup a new registry for each test..no need

Reply:
again added by Viraj. Already had reverted all of this on base branch